### PR TITLE
New set of 1-line art

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#130 
#### What does this implement/fix? Explain your changes.
- 97 new art:
	1. angry birds
	2. angry face2
  	3. angry troll
  	4. at what cost
  	5. barf
  	6. basking in glory
  	7. bear GTFO
	8. bear squiting
	9. boxing
	10. breakdown
	11. careless
	12. chasing
	13. cheers
	14. chicken
	15. confused scratch
	16. crazy
	17. creeper
	18. cry face
	19. cry troll
	20. cute face
	21. dance2
	22. dancing
	23. dear god why
	24. devious smile
	25. disagree
	26. discombobulated
	27. dislike1
	28. dislike2
	29. do you even lift bro?
	30. double flip
	31. dude glasses2
	32. emo dance
	33. excited
	34. exorcism
	35. eye roll
	36. feel perky
	37. fido
	38. fight
	39. fisticuffs
	40. flip friend
	41. fly away
	42. flying
	43. fuck you2
	44. happy face
	45. happy hug
	46. hitchhicking
	47. hugger
	48. im a hugger
	49. injured
	50. innocent face
	51. japanese lion face
	52. judgemental
	53. judging
	54. kissing
	55. kitty emote
	56. listening to headphones
	57. looking down
	58. love3
	59. meow
	60. no support
	61. opera
	62. peepers
	63. pretty eyes
	64. put the table back
	65. resting my eyes
	66. robot3
	67. running
	68. sad and confused
	69. sad and crying
	70. sad face
	71. satisfied
	72. seal
	73. shark face
	74. shy
	75. sleepy
	76. staring
	77. stranger danger
	78. strut
	79. stunna shades
	80. surprised3
	81. taking a dump
	82. tgif
	83. things that can_t be unseen
	84. tidy up
	85. tired
	86. touchy feely
	87. tripping out
	88. trolling
	89. wave dance
	90. whisling
	91. winnie the pooh
	92. winning
	93. wizard2
	94. worried
	95. yun
	96. zombie2
	97. zoned

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Hi
It seems there are some duplicate keys in `art_dic`:
1. suprised2
2. angry face
3. fuck you
Please take a look at codefactor issues ;-)

**Comment by sepandhaghighi** [comment]
@sadrasabouri Also it seems `point` and `pointing` are duplicated ;-)

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <a href="https://badge.fury.io/py/art"><img src="https://badge.fury.io/py/art.svg" alt="PyPI version" height="18"></a>
 <a href="https://www.python.org/"><img src="https://img.shields.io/badge/built%20with-Python3-green.svg" alt="built with Python3" /></a>
 <a href="https://github.com/sepandhaghighi/art/blob/master/FontList.ipynb"><img src="https://img.shields.io/badge/Font List-560-blue.svg"></a>
-<a href="https://github.com/sepandhaghighi/art/blob/master/ArtList.ipynb"><img src="https://img.shields.io/badge/Art List-499-orange.svg"></a>
+<a href="https://github.com/sepandhaghighi/art/blob/master/ArtList.ipynb"><img src="https://img.shields.io/badge/Art List-596-orange.svg"></a>
```
Comment (sepandhaghighi) [inline README.md:15]: 596 or 595 ?

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ ART is a Python lib for text converting to ASCII art fancy. ;-)
 	</tr>
 	<tr>
 		<td align="center">1-Line-Art Counter</td>
-		<td align="center">499</td>
+		<td align="center">596</td>
```
Comment (sepandhaghighi) [inline README.md:75]: 596 or 595 ?

File: art/test2.py [hunk 3]
```diff
@@ -579,184 +579,241 @@
...
+angry troll
+ヽ༼ ಠ益ಠ ༽ﾉ 
 ******************************
 angry2
-( ͠° ͟ʖ ͡°)﻿
+( ͠° ͟ʖ ͡°)﻿ 
 ******************************
 ankush
-︻デ┳═ー*----*
+︻デ┳═ー*----*
```
Comment (sepandhaghighi) [inline art/test2.py:639]: please remove this trailing white spaces ;-)

**Comment by sepandhaghighi** [comment]
@sadrasabouri Hi
I tried to fix all issues related to this PR.
Please take a look at my commits and leave your comments

**Comment by sepandhaghighi** [comment]
@ssheikholeslami Please review this PR ;-)

**Notice 1** : We have enough time
**Notice 2**: If everything is okay, please merge it after approval

**Comment by sepandhaghighi** [comment]
@ssheikholeslami 
⏰ Just a friendly reminder 😉

**Comment by sepandhaghighi** [comment]
@ssheikholeslami
⏰ Just a friendly reminder 😉

**Comment by sepandhaghighi** [comment]
@ssheikholeslami
⏰ Just another friendly reminder 😉

**Comment by sepandhaghighi** [comment]
@ssheikholeslami
⏰ Just another friendly reminder 😉

**Comment by sepandhaghighi** [comment]
@ssheikholeslami
⏰ Just another friendly reminder 😉

File: otherfile/font_check.py [hunk 4]
```diff
@@ -2,6 +2,8 @@
 """Fonts height, duplication and UTF-8 compatibility testing script."""
 import sys
 import art
+from art.art_param import NON_ASCII_FONTS
+from font_wizard import is_utf8,is_ascii
```
Comment (ssheikholeslami) [inline otherfile/font_check.py:6]: should be `from font_wizard import is_utf8, is_ascii`

**Review by ssheikholeslami — CHANGES_REQUESTED** [review]

**Review by ssheikholeslami — APPROVED** [review]

**Pull request merged**